internal/cephfs/controllerserver.go

```diff
@@ -361,7 +361,7 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 
 	RoundOffSize := util.RoundOffBytes(req.GetCapacityRange().GetRequiredBytes())
 
-	if err = createVolume(ctx, volOptions, cr, volumeID(volIdentifier.FsSubvolName), RoundOffSize); err != nil {
+	if err = resizeVolume(ctx, volOptions, cr, volumeID(volIdentifier.FsSubvolName), RoundOffSize); err != nil {
 		klog.Errorf(util.Log(ctx, "failed to expand volume %s: %v"), volumeID(volIdentifier.FsSubvolName), err)
```

Comment:
Is 'expand' actually correct here? Can CephFS not shrink volumes?

Reply:
it's supported in cephfs. but it's not supported in kubernetes word, you cannot shrink the PVC

Reply:
yeah, I thought so... but it may get added to CSI at one point? Bu I guess we'll just fix this up when we're testing that.